tests/modules/s3.py

```diff
@@ -45,7 +47,7 @@ def __init__(self, context: ContextT, bucket: Optional[str] = None) -> None:
         self._s3_bucket_name = bucket if bucket else config["bucket"]
 
         for module_logger in ("boto3", "botocore", "s3transfer", "urllib3"):
-            logging.getLogger(module_logger).setLevel(logging.CRITICAL)
+            logging.set_module_log_level(module_logger, logging.logging.CRITICAL)
```

Comment:
`logging.logging.CRITICAL` looks strange (double logging). Suggest to make CRITICAL available directly in `ch_tools.common.logging` namespace that it can be used as `logging.CRITICAL`.

The same is applicable for other log level constants and `getHandler` function.